### DISCUSSION TITLE
Zellij for Microsoft Windows?

### DISCUSSION THREAD

**Discussion by i2** [zellij-org/zellij#2109] [General]
First of all, thanks for creating Zellij.

Now I know this might sound ridiculous, but I was wondering if it would be possible to extend Zellij for Microsoft Windows. I mean not for WSL but proper Windows. There is a project called [vtm](https://github.com/netxs-group/vtm) that sets to do something similar.

Thank you!

**Comment by imsnif** [comment]
Definitely in the cards!

**Comment by i2** [comment]
And I know ETA questions are least favorite to hear (if not annoying :D) for any software maintainer. Just curious, is this a near-term goal or long-term wish-list or ...? Thanks!

**Comment by imsnif** [comment]
Long-term wish-list, for sure. :) Just for starters, I have no access to an MS windows machine and no budget to change that. Even if I did though, this is a monumental task and there are many other things ahead of it.
